### PR TITLE
Manually bump in-toto-golang to v0.8.0

### PR DESCRIPTION
**Pull request by chitrangpatel**
`in-toto-golang` introduced support for SLSA1.0 predicate in v0.8.0 This PR manually bumps `in-toto-golang` to v0.8.0.
This PR partly addresses issue https://github.com/tektoncd/chains/issues/728.

# Changes

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [x] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [x] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

``` release-note
NONE
```

### CODE REVIEW

File: pkg/config/zz_generated.deepcopy.go [hunk 1]
```diff
@@ -51,8 +51,9 @@ func (in *Artifact) DeepCopy() *Artifact {
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
 func (in *ArtifactConfigs) DeepCopyInto(out *ArtifactConfigs) {
 	*out = *in
-	in.TaskRuns.DeepCopyInto(&out.TaskRuns)
 	in.OCI.DeepCopyInto(&out.OCI)
+	in.PipelineRuns.DeepCopyInto(&out.PipelineRuns)
```
Comment (chuangw6) [inline pkg/config/zz_generated.deepcopy.go:55]: why is this file needed to be updated?

Comment (chitrangpatel) [reply pkg/config/zz_generated.deepcopy.go]: fixed.

**Comment by wlynch** [comment]
third_party directory is causing trouble - we removed this awhile back, but looks like it was never removed from hack/update_deps.sh. https://github.com/tektoncd/chains/pull/796 to fix.

**Comment by wlynch** [comment]
For now you can just `rm -rf third_party` and this PR should be good to go.

**Comment by chitrangpatel** [comment]
> For now you can just `rm -rf third_party` and this PR should be good to go.

Thanks! Done.

**Comment by chitrangpatel** [comment]
ping for a quick review @wlynch 🙏 Thanks!
